scripts/workspace_test.py

```diff
@@ -31,6 +31,7 @@
     args = parser.parse_args(sys.argv[1:])
 
     VERBOSE = args.verbose
+    receivedOutputJson = []
```

Comment:
I would move this line after the line 
https://github.com/IBM/watson-assistant-workbench/blob/c46baa46bc8526a02b33b647a9c4bde9c1ad01af/scripts/workspace_test.py#L76